packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/base/repositories/users_repository.dart

```diff
@@ -0,0 +1,60 @@
+{{> licence.dart }}
+
+import '../common_mappers/error_mappers/error_mapper.dart';
+import '../data_sources/local/users_local_data_source.dart';
+import '../data_sources/remote/users_remote_data_source.dart';
+import '../models/credentials_model.dart';
+import '../models/request_models/confirm_email_model.dart';
+import '../models/user_model.dart';
+import '../models/user_with_auth_token_model.dart';
+
+class UsersRepository {
```

Comment:
While most of the methods behaviour can be intuitively inferred from their name, it would benefit for some of them to have them documented (for instance: `getMyUser`, `clearIsProfileTemporary ` and similar as they seem to be specific to this domain).